### PR TITLE
Issue #53 - Support for <RIGHT> alignment tag on basic HTML

### PR DESCRIPTION
**Pull request by gpascual2**
Updated the HTML basic parser to support the \<RIGHT\> and \<LEFT\> tags along with the \<CENTER\> one that was already supported. Left is pretty simple since is the default fallback alignemnt. The real benefit of this change is the support for Right alignment. 

I also added a line with \<right\> allignment on the fpdf_test suite for the ExampleFpdf_HTMLBasicNew(). 

Test result is attached here: 
[Fpdf_HTMLBasicNew.pdf](https://github.com/jung-kurt/gofpdf/files/626228/Fpdf_HTMLBasicNew.pdf)

This change is related to Issue #53.

### CODE REVIEW

File: fpdf_test.go [hunk 1]
```diff
@@ -470,6 +470,7 @@ func ExampleFpdf_HTMLBasicNew() {
 	htmlStr := `You can now easily print text mixing different styles: <b>bold</b>, ` +
 		`<i>italic</i>, <u>underlined</u>, or <b><i><u>all at once</u></i></b>!<br><br>` +
 		`<center>You can also center text.</center>` +
+		`<right>Or align it to the right.</right>` +
```
Comment (gpascual2) [inline fpdf_test.go]: Adding a test line for Right text alignment.

File: htmlbasic.go [hunk 2]
```diff
@@ -169,7 +169,7 @@ func (html *HTMLBasicType) Write(lineHt float64, htmlStr string) {
 				putLink(hrefStr, el.Str)
 				hrefStr = ""
 			} else {
-				if alignStr == "C" {
+				if alignStr == "C" || alignStr == "R" {
```
Comment (gpascual2) [inline htmlbasic.go]: Use the WriteAligned function also for 'R'ight.

File: htmlbasic.go [hunk 3]
```diff
@@ -188,6 +188,12 @@ func (html *HTMLBasicType) Write(lineHt float64, htmlStr string) {
 			case "center":
 				html.pdf.Ln(lineHt)
 				alignStr = "C"
+			case "right":
```
Comment (gpascual2) [inline htmlbasic.go]: Adding support for 'right' and 'left' opening html tags. Mapped the right tag with the 'R' alignment of the pdf.WriteAligned function.

File: htmlbasic.go [hunk 4]
```diff
@@ -205,6 +211,9 @@ func (html *HTMLBasicType) Write(lineHt float64, htmlStr string) {
 			case "center":
 				html.pdf.Ln(lineHt)
 				alignStr = "L"
+			case "right":
```
Comment (gpascual2) [inline htmlbasic.go]: Adding support for the 'right' closing html tag. Fallbacking to 'left' alignment after close.

**Comment by jung-kurt** [comment]
Thanks, @gpascual2! Nice feature.

**Pull request merged**